src/group_theory/group_action/sigma.lean

```diff
@@ -0,0 +1,52 @@
+/-
+Copyright (c) 2022 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import group_theory.group_action.defs
+
+/-!
+# Sigma instances for additive and multiplicative actions
+
+This file defines instances for arbitrary sum of additive and multiplicative actions.
```

Comment:
I think a " see also `sum.has_scalar` " and vice versa in the other file would be sensible.